src/Tizen.NUI.Components/Controls/ButtonGroup.cs

```diff
@@ -27,7 +27,7 @@ namespace Tizen.NUI.Components
     /// </summary>
     /// <since_tizen> 3 </since_tizen>
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class ButtonGroup : BindableObject, global::System.IDisposable
+    public class ButtonGroup : View
```

Comment:
Can you explain why did you change inheritancy of ButtonGroup in this patch?
I think it is not related this patch.

Reply:
This modification is related to Layout. I will remove it from this patch